boards/nxp/frdm_mcxa153/doc/index.rst

```diff
@@ -0,0 +1,206 @@
+.. zephyr:board:: frdm_mcxa153
+
+Overview
+********
+
+FRDM-MCXA153 are compact and scalable development boards for rapid prototyping of
+MCX A14 and A15 MCUs. They offer industry standard headers for easy access to the
+MCU's I/Os, integrated open-standard serial interfaces, external flash memory and
+an on-board MCU-Link debugger. Additional tools like our Expansion Board Hub for
+add-on boards and the Application Code Hub for software examples are available
+through the MCUXpresso Developer Experience.
+
+Hardware
+********
+
+- MCX-A153 Arm Cortex-M33 microcontroller running at 96 MHz
+- 128KB dual-bank on chip Flash
+- 32 KB RAM
+- USB full-speed with on-chip FS PHY. USB Type-C connectors
+- 1x I3Cs
+- On-board MCU-Link debugger with CMSIS-DAP
+- Arduino Header, mikroBUS
+
+For more information about the MCX-A153 SoC and FRDM-MCXA153 board, see:
+
+- `MCX-A153 SoC Website`_
+- `MCX-A153 Datasheet`_
+- `MCX-A153 Reference Manual`_
+- `FRDM-MCXA153 Website`_
+- `FRDM-MCXA153 User Guide`_
+- `FRDM-MCXA153 Board User Manual`_
+- `FRDM-MCXA153 Schematics`_
+
+Supported Features
+==================
+
+The ``frdm_mcxa153`` board supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
```

Comment:
As @nordicjm shared, these Supported Features tables are auto-generated now:
```suggestion
.. zephyr:board-supported-hw::
```

Reply:
> As @nordicjm shared, these Supported Features tables are auto-generated now:

We should probably add it for all NXP platforms.

Reply:
> > As @nordicjm shared, these Supported Features tables are auto-generated now:
> 
> We should probably add it for all NXP platforms.

already been done

Reply:
> > > As @nordicjm shared, these Supported Features tables are auto-generated now:
> > 
> > 
> > We should probably add it for all NXP platforms.
> 
> already been done

Missed this moment. Great!